### PR TITLE
Add Nomic and Multilingual Cohere

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

### Adding a model checklist

 - [x] I have filled out the ModelMeta object to the extent possible
 - [x] I have ensured that my model can be loaded using
   - [x] `mteb.get_model(model_name, revision_id)` and
   - [x] `mteb.get_model_meta(model_name, revision_id)`
 - [x] I have tested the implementation works on a representative set of tasks.
  - Implementation of Cohere match with SEB for AngryTweets

### CODE REVIEW

File: mteb/evaluation/evaluators/model_encode.py [hunk 1]
```diff
@@ -15,12 +15,16 @@ def model_encode(
...
-    if hasattr(model, "prompts") and prompt_name not in model.prompts:  # type: ignore
-        logger.info(
-            f"Prompt {prompt_name} not found in model prompts. Removing prompt_name argument."
-        )
-        kwargs.pop("prompt_name")
-
+    if hasattr(model, "prompts"):
+        # check if prompts is an empty dict
+        if not model.prompts:  # type: ignore
+            logger.info("Model does support prompts. Removing prompt_name argument.")
```
Comment (Muennighoff) [inline mteb/evaluation/evaluators/model_encode.py:21]:
```suggestion
            logger.info("Model does not support prompts. Removing prompt_name argument.")
```

File: mteb/models/nomic_models.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+from mteb.model_meta import ModelMeta, sentence_transformers_loader
+
+nomic_embed = ModelMeta(
+    loader=partial(
+        sentence_transformers_loader,
+        trust_remote_code=True,
+        model_name="nomic-ai/nomic-embed-text-v1.5",
+        revision=None,
+    ),
```
Comment (Muennighoff) [inline mteb/models/nomic_models.py:11]: Do their models not require some extra normalization?

See https://huggingface.co/nomic-ai/nomic-embed-text-v1.5#sentence-transformers

**Review by Muennighoff — COMMENTED** [review]
Looks great, minor points!

File: mteb/models/nomic_models.py (hunk 2 above)
Comment (KennethEnevoldsen) [reply mteb/models/nomic_models.py]: I have added this in the latest PR

**Review by Muennighoff — APPROVED** [review]

**Pull request merged**